### ISSUE TITLE
Change facet options sort order if not by count

### ISSUE THREAD

**Issue by jrchudy** [informatics-isi-edu/ermrestjs#884]
When counts are suppressed, facet options need to be sorted some other way (related to issue #883 ). Maybe this should be something that is configurable.

**Comment by RFSH** [comment]
This feature has been implemented by #694 PR (original issue: #690) and is documented in [here](https://github.com/informatics-isi-edu/ermrestjs/blob/master/docs/user-docs/facet-json-structure.md#order). 

It seems like when I implemented this feature I forgot to properly document it in the annotation document, so I did it as part of https://github.com/informatics-isi-edu/ermrestjs/commit/31b765b158aa356258c0f2b582a815994eacd59b commit. 

To summarize, in order to remove the count request from the scalar facets we would have to hide the option and also change the sort so it's not based on "number of occurrences". So for example, 

```json
{
    "source": [{"outbound": ["s", "fk1_cons"]}, "f1_text"],
    "entity": false,
    "hide_num_occurrences": true,
    "order": [
        {"column": "f1_text", "descending": false}
    ]
}
```
